components/errors/src/main/kotlin/org/cloudfoundry/credhub/exceptions/InvalidAlternateNameCertificateException.kt

```diff
@@ -0,0 +1,8 @@
+package org.cloudfoundry.credhub.exceptions
+
+import org.cloudfoundry.credhub.ErrorMessages
+
+class InvalidAlternateNameCertificateException @JvmOverloads
+    constructor(messageCode: String = ErrorMessages.INVALID_ALTERNATE_NAME, parameters: Array<Any> = arrayOf())
```

Comment:
It feels like now the error message constant could and should be moved to this exception class.